package.json

```diff
@@ -41,18 +41,18 @@
   "dependencies": {
     "brout": "^1.3.0",
     "listen": "^1.0.0",
-    "mocha": "^8.4.0",
+    "mocha": "^9.1.0",
     "resolve": "^1.8.1",
-    "supports-color": "^5.5.0",
-    "through2": "^2.0.5"
+    "supports-color": "^8.1.1",
+    "through2": "^4.0.2"
   },
   "devDependencies": {
     "@studio/changes": "^2.2.0",
-    "browserify": "^16.2.3",
+    "browserify": "^17.0.0",
```

Comment:
I'm wondering if we should stick to 16 here so that it's in sync with the (blocked) Mochify: https://github.com/mantoni/mochify.js/issues/224

Reply:
I removed the browserify update